api/src/main/java/org/apache/iceberg/Scan.java

```diff
@@ -137,6 +144,14 @@
    */
   CloseableIterable<G> planTasks();
 
+  /**
+   * Returns whether this scan should apply column name case sensitiveness as per {@link
+   * Scan#caseSensitive(boolean)}.
```

Comment:
Nit: maybe `Return whether this scan should be case sensitive with respect to column names` or something else that avoids the phrase `apply column name case sensitiveness`. Even just avoiding `case sensitiveness` would help me to think less about it when looking through it.

EDiT - I see this is the same wording carried from `TableScan`. Would not advise changing for this reason alone then.

Reply:
I copied the doc from `TableScan` but I'd be up for updating it.

Reply:
Updated.